com.insynchq.Insync.yml

```diff
@@ -0,0 +1,51 @@
+app-id: com.insynchq.Insync
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+base-version: "24.08"
+command: /app/usr/lib/insync/insync
```

Comment:
command is NEVER a full path. and also it MUST match the `Exec` in the desktop (minus arguments)